arithmetization/src/main/java/net/consensys/linea/zktracer/module/mmu/instructions/InvalidCodePrefix.java

```diff
@@ -77,14 +76,12 @@ public MmuData preProcess(MmuData mmuData) {
             .remainder(rem)
             .build());
 
-    microLimb =
-        Bytes.of(
-            mmuData
-                .sourceRamBytes()
-                .get((int) (LLARGE * initialSourceLimbOffset + initialSourceByteOffset)));
-    Bytes arg1 = microLimb;
-    Bytes arg2 = Bytes.of(EIP_3541_MARKER);
-    boolean result = wcp.callEQ(arg1, arg2);
+    final int offset = (int) (LLARGE * initialSourceLimbOffset + initialSourceByteOffset);
+    final boolean offsetOutOfBond = offset >= mmuData.sourceRamBytes().size();
+    microLimb = offsetOutOfBond ? Bytes.of(0) : Bytes.of(mmuData.sourceRamBytes().get(offset));
```

Comment:
The only thing that matters, the rest is cosmetic

Reply:
Do we not have direct access to the value of `offset` without recomputing it as `LLARGE * initialSourceLimbOffset + initialSourceByteOffset` ?

Reply:
`offsetOutOfBond` should be `offsetIsOutOfBounds`

Reply:
Do we not have direct access to the value of offset without recomputing it as LLARGE * initialSourceLimbOffset + initialSourceByteOffset ?

Reply:
offsetOutOfBond should be offsetIsOutOfBounds

Reply:
Yes & Yes